sw/device/lib/crypto/drivers/entropy.c

```diff
@@ -104,6 +104,20 @@ static status_t csrng_send_app_cmd(uint32_t reg_address,
   for (size_t i = 0; i < cmd_len; ++i) {
     abs_mmio_write32(reg_address, cmd.seed_material->data[i]);
   }
+
+  // Poll the "command request done" interrupt bit. Once it is set, this
+  // signals that the command has been processed and the "status" bit is
+  // updated.
+  do {
+    reg = abs_mmio_read32(kBaseCsrng + CSRNG_INTR_STATE_REG_OFFSET);
+  } while (!bitfield_bit32_read(reg, CSRNG_INTR_STATE_CS_CMD_REQ_DONE_BIT));
```

Comment:
Do we need to clear the `cs_cmd_req_done` bit (by writing `1` to it) after this polling, so that the next invocation of this function does not mistake the latched interrupt as "another command request is complete"?

Reply:
Ui, yes, good catch. I was thinking about that, too but forgot to check when doing the review. The bit should probably be cleared also before writing the first command in this driver.